AK/Optional.h

```diff
@@ -448,6 +448,14 @@ requires(IsLvalueReference<T>) class [[nodiscard]] Optional<T> {
         return fallback;
     }
 
+    template<typename U>
+    [[nodiscard]] ALWAYS_INLINE Optional<U> map(Function<Optional<U>(RemoveCVReference<T>)> mapping)
```

Comment:
Just noticed this, this exists in master now, might wanna remove this overload

Reply:
who sniped me to this?

Reply:
I added Optional::map() for jakt a while back, we merged that back into AK like...last week?
that one even supports TRY() lol